slackbot/dispatcher.py

```diff
@@ -17,10 +17,11 @@
 
 
 class MessageDispatcher(object):
-    def __init__(self, slackclient, plugins):
+    def __init__(self, slackclient, plugins, default_reply=None):
         self._client = slackclient
         self._pool = WorkerPool(self.dispatch_msg)
         self._plugins = plugins
+        self._override_reply = default_reply
```

Comment:
There is a funciton called _default_reply already. It would override it. We can change the name ot he functino if you prefer ! :grin: